### DISCUSSION TITLE
cfde-submit status related

### DISCUSSION THREAD

**Discussion by mano-at-sdsc** [nih-cfde/published-documentation#363] [Bug] [answered]
I am getting the following message:

(base) [mano@sc-ngomics C2M2_sub]$ cfde-submit status

Status of CFDE Submission (Flow ID 8fd3f20d-a670-44e5-88fd-7d0efad8ac09)
This instance ID: 54f01a93-207c-4ecd-9810-313f8b26cc63

This flow has failed.
{
    "cause": {
        "action_id": "b63cd06b-353e-4692-a1ad-b02988950a2f",
        "display_status": "Function Active",
        "message": "Timeout on state FairAssessment. Max time 300; Elapsed time 570.5913827419281",
        "status": "ACTIVE"
    },
    "exception": "ActionTimeout",
    "state_name": "FairAssessment"
}

**Comment by mano-at-sdsc** [comment]
It appears the submission was successful. I see this on the portal:

![image](https://user-images.githubusercontent.com/72581953/163486709-c06dca6c-ac71-40bf-a85c-5e2f41ce06c1.png)

I will review the datapackage.

**Comment by DavidMichaelKelly** [comment, marked as answer]
The should be resolved now. It looks like your most recent submission (submission id be6602ac-bcf4-11ec-a325-5376c1e42e63) contains the correct metrics data and was approved. Thank you for the report!
